src/main/java/be/seeseemelk/mockbukkit/WorldMock.java

```
@@ -684,6 +754,12 @@ public Collection<Entity> getEntitiesByClasses(Class<?>... classes)
 		throw new UnimplementedOperationException();
 	}
 
+	@Override
+	public @NotNull CompletableFuture<Chunk> getChunkAtAsync(int x, int z, boolean gen, boolean urgent)
+	{
+		return null;
```

This should throw an exception